homeworks/dmytro.shevchenko_dmytro-shevchenko1/GamesProject/login.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Login Page</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+    <body class="login-page">
+        <header>
```

Comment:
It's better to apply styles using a class. Please add a class here (e.g., class="header") and apply the styles to it